src/main/java/com/springbootassignment/cruddemo/controller/CourseController.java

```diff
@@ -0,0 +1,65 @@
+package com.springbootassignment.cruddemo.controller;
+
+import com.springbootassignment.cruddemo.entity.Course;
+import com.springbootassignment.cruddemo.entity.Student;
+import com.springbootassignment.cruddemo.service.CourseService;
+import com.springbootassignment.cruddemo.service.StudentService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
+@RequestMapping("/courses")
+public class CourseController {
+
+    private CourseService courseService;
+
+    @Autowired
```

Comment:
You can directly use the @Autowired annotation above the decalred service variable insted of manually initializing it.